llvm/lib/Transforms/Vectorize/SandboxVectorizer/SandboxVectorizer.cpp

```diff
@@ -0,0 +1,39 @@
+//===- SandboxVectorizer.cpp - Vectorizer based on Sandbox IR -------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "llvm/Transforms/Vectorize/SandboxVectorizer/SandboxVectorizer.h"
+#include "llvm/Support/CommandLine.h"
+
+using namespace llvm;
+
+#define SV_NAME "sandbox-vectorizer"
+#define DEBUG_TYPE "SBVec"
```

Comment:
`SBVectorizer`?

Reply:
We could change it to `#define DEBUG_TYPE SV_NAME` like it's done in LoopVectorize.cpp. Wdyt?

Reply:
SGTM.